src/Apply.hs

```diff
@@ -80,7 +80,7 @@ parseModuleApply flags s file src = do
     res <- parseModuleEx (parseFlagsAddFixities [x | Infix x <- s] flags) file src
     case res of
         Right (ParsedModuleResults (m, c) _)  -> return $ Right (m,c)
-        Left (ParseError sl msg ctxt _) ->
+        Left (ParseError sl msg ctxt) ->
             return $ Left $ classify [x | SettingClassify x <- s] $ rawIdeaN Error "Parse error" (mkSrcSpan sl sl) ctxt Nothing []
```

Comment:
Is this where the user error message is formed? Sure enough the `msg` is not used here and the result is consistent with what I observe testing ('`Parse error`').